sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -1431,4 +1431,30 @@ class HiveDDLSuite
       }
     }
   }
+
+  test("insert data to a table which has altered the table location " +
+    "to an not exist location should success") {
```

Comment:
Test case names are not accurate after you add new test cases.  Actually, could you split the test cases?